cmd/api/api.go

```diff
@@ -74,6 +69,14 @@ func NewAPICmd() *cobra.Command {
 }
 
 func (c *CmdOpts) startAPI() error {
+	// get k0s config
+	loadingRules := config.ClientConfigLoadingRules{Nodeconfig: true}
```

Comment:
can this be part of the GetCmdOpts function in github.com/k0sproject/k0s/pkg/config package, which is used as contruction function for CmdOptions?
It will allow to drop repetative calls of loading.Rules.Load everywhere.